app/controllers/dashboard_controller.rb

```diff
@@ -1,6 +1,6 @@
 class DashboardController < ApplicationController
   def show
-    @profile = current_user.profile || current_user.create_profile
```

Comment:
This had to be cut because `create_profile` is a private method and can't be called like this. Also in practice a user should never not have a profile. This is backed up and supported in the User model, where create_profile is being called in an after_create_commit callback.